### DISCUSSION TITLE
How to adjust mqtt message update frequency

### DISCUSSION THREAD

**Discussion by maxmib** [Koenkk/zigbee2mqtt#13522] [General]
I'm looking for the way to change mqtt message update frequency in zigbee2mqtt.For example,my tuya tvoc detector reports per second,but I want zigbee2mqtt to send detector's message per minute.How can I change the mqtt message update frequency or my tvoc detector's report interval ?Thanks.

**Comment by sjorge** [comment]
zigbee2mqtt just publishes when the sensor sends data, you might be able to update the device's reporting minimum interval. But this does not work for a lot of devices like TuYa and Aqara.

Alternatively you can set a `debounce: 60` for the device but that just holds off on publishing for 60 seconds.

**Reply by maxmib** [reply]
after setting  debouce：5 ，zigbee2mqtt stops publishing data that received from tuya detector. Because detector sends different data per second,data can't  stay the same for 5 seconds.
